docs/releases/5.2.txt

```diff
@@ -105,6 +105,13 @@ Minor features
   processing instruction will be added to the top of the document for each
   stylesheet in the given list. See :ref:`feed-stylesheets` for more details.
 
+:mod:`django.shortcuts`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* :func:`~django.shortcuts.redirect` now supports the boolean argument
+  ``preserve_method`` which signals the user agent to preserve the HTTP method
+  when doing the redirection.
```

Comment:
We can summarize this like this:

```suggestion
* :func:`~django.shortcuts.redirect` now supports preserving the HTTP method
   during redirection.
```

And then we add further details to the `redirect` docs. We should also move this entry to `Requests and Responses`.